src/responder/kcm/kcmsrv_ccache.c

```diff
@@ -273,6 +273,21 @@ static krb5_creds *kcm_cred_to_krb5(krb5_context kctx, struct kcm_cred *kcm_crd)
 }
 #endif
 
+static void securely_erase_krb5_data(krb5_data *data)
+{
+    if (data != NULL) {
+        sss_erase_mem_securely(data->data, data->length);
+    }
+}
+
+static void securely_erase_krb5_creds(krb5_creds *cred)
+{
+    if (cred != NULL) {
+        securely_erase_krb5_data(&cred->ticket);
+        securely_erase_krb5_data(&cred->second_ticket);
```

Comment:
I would include `krb5_free_creds` here (optional).
There is nothing to do with erased structures anyway.

Reply:
> There is also `krb5_authdata ** krb5_creds.authdata`. What does it contain?

I tried to understand what this array contains, but found no information. The name seem to indicate it is data about the authorizations, so I guess this is no secret to be cleaned. Because cleaning takes some time, and it is done many times, I tried to stick to the bare minimum. If you think this MUST be cleaned, I'll add it.